aries_cloudagent/anoncreds/anoncreds/anoncreds_registry.py

```diff
@@ -87,6 +97,16 @@ async def get_credential_definition(
         raise AnonCredsObjectNotFound(
             f"{credential_definition_id} could not be resolved"
         )
+    
+    async def get_credential_definitions(
+        self, filter : dict
+    ):
+        """Get credential definitions id's from the registry."""
+        results = [
+            resolver.get(filter)
```

Comment:
Typo? `resolver.get(filter)` perhaps should be `resolver.get_credential_definitions`?

Reply:
good catch!